web/src/routes/admin/+page.svelte

```diff
@@ -1,2 +1,118 @@
 <script lang="ts">
+  import Card from '$lib/components/ui/card/card.svelte';
+  import { Icons } from '$lib/utils';
+  import LinkBar from '$lib/components/ui/link-bar/link-bar.svelte';
+  import DataTable, { type DataItem } from '$lib/elements/ui/dataTable/data-table.svelte';
+  import {
+    deleteTeacher,
+    getServerStats,
+    getTeachers,
+    type Pageable,
+    type PageTeacher,
+    type ServerStatisticsResponseDto,
+  } from '$lib/sdk/fetch-client';
+  import { error } from '@sveltejs/kit';
+  import { onMount } from 'svelte';
+  import * as Select from '$lib/elements/ui/select';
+
+  let columnNames = ['Vorname', 'Nachname', 'Akronym', 'Fächer', 'Tags'];
+  let keys = ['id', 'firstName', 'lastName', 'acronym', 'subjects', 'tags'];
+  let pageLoaded = false;
+
+  const settings = [
+    { value: 'edit_constraints', label: 'Constraints ändern', url: '/admin/constraints' },
+    { value: 'generate_timetable', label: 'Stundenplan generieren', url: '/admin/timetable' },
+    { value: 'edit_curriculum', label: 'Curriculum anpassen', url: '/admin/curriculums' },
+    { value: 'edit_dsl', label: 'DSL Editor', url: '/admin/editor' },
+    { value: 'general_settings', label: 'Allgemeine Einstell.', url: '/admin/settings' },
+  ];
+
+  let stats: ServerStatisticsResponseDto;
+
+  onMount(async () => {
+    pageLoaded = true;
+    stats = await loadStats();
+  });
+
+  async function loadStats() {
+    const response = await getServerStats();
+    return response;
+  }
```

Comment:
You should wrap the affected html (the part that requires the response) in `{# await getServerStats() then stats}` and work with that.

Reply:
thank you, done